framework/sandstone_context_dump.cpp

```diff
@@ -465,6 +465,9 @@ void dump_xsave(FILE *f, const void *xsave_area, size_t xsave_size, int xsave_du
 
         if (xsave_bv & ~xgetbv0)
             return;     // bit vector contains invalid bits
+    } else {
```

Comment:
So this is the case of `xsave_size == Fxsave::size`?

Reply:
Yes.